### ISSUE TITLE
In Mac Scrollbar is not visible in SAP Horizon High Contrast Black theme

### ISSUE THREAD

**Issue by vsomsekhar** [SAP/fundamental-ngx#10978]
Is this a bug, enhancement, or feature request?
bug

In SAP Horizon High Contrast Black the scrollbar colour is changing to black and background is also black so the scrollbar is not visible.

Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

"@fundamental-ngx/core": "0.46.1"
"@angular/core": "^16.2.8"

If this is a bug, please provide steps for reproducing it; the exact components you are using;

SAP Horizon High Contrast Black (as scrollbar is black in colour, it is not visible)
<img width="1126" alt="Screenshot 2023-11-17 at 10 04 58 AM" src="https://github.com/SAP/fundamental-ngx/assets/142768234/6671ec99-431a-4356-a067-0a864da7a4a8">

SAP Morning Horizon
<img width="1119" alt="Screenshot 2023-11-17 at 10 04 04 AM" src="https://github.com/SAP/fundamental-ngx/assets/142768234/073e19b6-7654-4d56-81b7-2f9b2c1a85a9">

**Comment by droshev** [comment]
@vsomsekhar Can you create a simple stackblitz example?

**Comment by dhivya910** [comment]
@vsomsekhar please provide the required example to proceed with this issue

**Comment by SomVelisetty** [comment]
I am unable to get the same scene in StackBlitz.

**Comment by droshev** [comment]
It looks like there is something specific in your scenario. If you are able to provide any isolated app where we can reproduce the issue we can take a look. Please, report a new ticket if you can do so.
